aie_runtime_lib/AIE/erf.h

```diff
@@ -0,0 +1 @@
+// Unsupported erf.h for AIE1
```

Comment:
A minor comment. We can do the same thing to state for unsupported for  https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/sqrt.h, 
https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/tanh.cpp, and https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/tanh.h

Reply:
> A minor comment. We can do the same thing to state for unsupported for https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/sqrt.h, https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/tanh.cpp, and https://github.com/Xilinx/mlir-aie/blob/main/aie_runtime_lib/AIE/tanh.h

I will reorganize the files in the aie_runtime_lib in my next changeset.